Dockerfile.build

```diff
@@ -4,7 +4,7 @@ RUN dnf -y install sudo git cabal-install ostree-devel xz-devel \
                    gobject-introspection-devel libcurl-devel \
                    ghc-haskell-gi-devel libgit2-glib-devel which && \
     cabal update && \
-    cabal install --global happy hlint hpc-coveralls
+    cabal install --global 'aeson <= 1.3' happy hlint 'hpc-coveralls > 1.0'
```

Comment:
I really don't like doing this. I wonder if we could build these tools in their own images and then just copy the results instead of having their dependencies crap up the dependencies for everything else.